### ISSUE TITLE
SCI Spec - Submitted question

### ISSUE THREAD

**Issue by seanmcilroy29** [Green-Software-Foundation/sci#370]
From Hallie Cramer - Google

If the electricity consumption is connected to a grid, the short run marginal, long run marginal, or average emissions grid intensity of that grid shall be used" - it appears this is being used to refer to how to calculate an SCI score, which may be different than "carbon-aware: attribute of software or hardware that adjusts its behavior (consumption of inputs, processing, or production of outputs) in response to the carbon intensity of the energy it consumes". I bring this up as an area of confusion because the SCI score, a calculation of the carbon intensity of a software, I would interpret as "attributing" emissions to the software, for which it is very standard practice to use an average emissions factor. A marginal emissions factor is generally used as a proxy to measure the change in emissions with an incremental change in energy, which may be more relevant as a signal for being "carbon-aware", but would still say there is no consensus around which signal is best, so it makes sense to leave options.

**Comment by Henry-WattTime** [comment]
Hi Hallie, this pull request contains much of the thinking around what carbon signal we should use for the spec:
https://github.com/Green-Software-Foundation/sci/pull/357
Is this helpful? Would it make sense to discuss in more depth?

**Comment by halliecramer1** [comment]
Yes, that is helpful. It looks like the PR was closed - does that mean this will not be merged in anywhere? I think it will be important to add guidelines in the user guide which clarify why / when you would use each of the emissions rates that are mentioned in the standard, for more clarity on the strengths and weaknesses of each one - something like what is included in the thread. Is it possible to copy that language over in a new PR [here](https://github.com/Green-Software-Foundation/sci-guide/blob/web/docs/I/index.md)?
